README.md

```diff
@@ -17,11 +17,12 @@ class C extends HTMLElement {
 }
 ```
 
-Decorators have three primary capabilities:
+Decorators have four primary capabilities:
 
 1. They can **replace** the value that is being decorated with a _matching_ value that has the same semantics. (e.g. a decorator can replace a method with another method, a field with another field, a class with another class, and so on).
 2. They can associate **metadata** with the value that is being decorated. This metadata can then be read externally and used for metaprogramming and introspection.
 3. They can provide **access** to the value that is being decorated, via metadata. For public values, they can do this via the name of the value. For private values, they receive accessor functions which they can then choose to share.
+4. They can **initialize** the value that is being decorated, running additional code after the value has been fully defined. In cases where the value is a member of class, then initialization occurs once per instance.
```

Comment:
```suggestion
4. They can **initialize** the value that is being decorated, running additional code after the value has been fully defined. In cases where the value is a member of a class, then initialization occurs once per instance.
```

Reply:
Wasn't sure if you mean "member of a class" or "member or class". The first seemed right, so added an article for you.